Components.md

```diff
@@ -0,0 +1,137 @@
+# Core
```

Comment:
Thank you for using English in your materials. To be honest this way creates some difference in level of understanding for other students (Previous materials was prepared in Russian). 
May be create two versions of same materials (ru and en)?

Reply:
Yes, I added version of the same material in Russian